static/src/server.ts

```diff
@@ -538,9 +644,9 @@ app.get("/nodejs/query", (req: Request, res: Response) => {
 
       Promise.all(tilePromises)
         .then((tileResults) => {
-          const filteredResults = tileResults.filter(
-            (result) => result !== null
-          );
+          const filteredResults = tileResults
+            .filter((result) => result !== null)
+            .flat(1);
```

Comment:
Could any of the nested tile results be null? e.g., `[tileResult1, tileResult2, [tileResult3, null]]`- the filter might not catch these cases

Reply:
the nested tile results couldn't be null, but flipped the order of flattening and filtering in case future changes could cause nested results to be null!